### PR TITLE
Bump luakit

### PR DESCRIPTION
**Pull request by oz123**

### CODE REVIEW

File: www-client/luakit/luakit-2.3.1.ebuild [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```
Comment (ConiKost) [inline www-client/luakit/luakit-2.3.1.ebuild:4]: Please bump to EAPI=8.

Comment (oz123) [reply www-client/luakit/luakit-2.3.1.ebuild]: Thanks! I updated this and the EAPI as well.

File: www-client/luakit/luakit-2.3.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	test? (
+		$(lua_gen_cond_dep '
+			dev-lua/luassert[${LUA_USEDEP}]
+			dev-lua/luacheck[${LUA_USEDEP}]
+		')
+		x11-base/xorg-server[xvfb]
+	)
+"
+PATCHES=(
+	"${FILESDIR}"/${PN}-2.2.1-make.patch
```
Comment (juippis) [inline www-client/luakit/luakit-2.3.1.ebuild:51]: > * ERROR: www-client/luakit-2.3.1::gentoo failed (prepare phase):
> *   patch -p1  failed with /var/tmp/portage/www-client/luakit-2.3.1/files/luakit-2.2.1-make.patch

File: www-client/luakit/luakit-2.3.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-1 luajit )
```
Comment (juippis) [inline www-client/luakit/luakit-2.3.1.ebuild:6]: Any chance to bump the lua compat above 5.1?

Comment (oz123) [reply www-client/luakit/luakit-2.3.1.ebuild:6]: I will have to look into upstream code for that. Good chance that it's possible.

**Pull request closed without merging**